tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/files/S3StorageSpec.scala

```diff
@@ -164,6 +173,23 @@ class S3StorageSpec extends StorageSpec {
         response.status shouldEqual StatusCodes.BadRequest
       }
     }
+
+    "succeed using the configured default bucket when a bucket is not provided" in {
+      val defaultBucket = "mydefaultbucket"
+      val id            = genId()
+      val payload       = jsonContentOf(
+        "kg/storages/s3.json",
+        "storageId" -> s"https://bluebrain.github.io/nexus/vocabulary/$id"
+      ).mapObject(_.remove("bucket"))
+
+      val expectedStorage = storageResponse(projectRef, id, defaultBucket, "resources/read", "files/write")
+
+      for {
+        _ <- createBucket(defaultBucket)
+        _ <- storagesDsl.createStorage(payload, projectRef)
+        _ <- storagesDsl.checkStorageMetadata(projectRef, id, expectedStorage)
```

Comment:
Not a big deal but it would be clearer if the assertion was just on the bucket name